src/pages/resume/CLAUDE.md

```
@@ -0,0 +1,12 @@
+<claude-mem-context>
```

markdownlint-cli2 (0.20.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)